network/groups.go

```diff
@@ -173,34 +170,29 @@ func randomWeightIndex(weights []uint64, count int) (randomWeights []int) {
 	return
 }
 
-func selectMinWeightIndex(weights []uint64, usedWeights map[int]types.Empty) int {
-	minLocation := -1
-	currentMin := weights[0]
-	for i, weight := range weights {
-		// skip indexes that has been already used
-		if _, ok := usedWeights[i]; !ok {
-			if weight < currentMin || minLocation == -1 {
-				currentMin = weight
-				minLocation = i
-			}
+// endregion ///////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// region Accidental ///////////////////////////////////////////////////////////////////////////////////////////////////
+
+func GetAccidentalIssuers(network *Network) []*Peer {
+	peers := make([]*Peer, 0)
+	randomCount := 0
+	for i := 0; i < len(config.AccidentalMana); i++ {
```

Comment:
Don't we also want to select issuers from a given index, sorted by the Weight? That means we could get a node from the middle of the distribution and use that as well.

Reply:
why not, might be usefull in the future, I'll add that